### ISSUE TITLE
Need strict mode for debounce.js?

### ISSUE THREAD

**Issue by clarkware** [tomekwi/elm-live#92]
I upgraded from 2.6.0 to 2.6.1 and started getting the following error:

/usr/local/lib/node_modules/elm-live/source/debounce.js:4
  let timeout;
  ^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode

I appears that debounce.js was added in 2.6.1.

**Comment by tomekwi** [comment]
Hey @clarkware, thanks a lot for the report! May I ask what version of node you’re running on? After merging this change, I also [updated the heads-up note](https://github.com/tomekwi/elm-live/commit/9d337675596a9c4655fabe8176627e8fa0a3c358) – you now need node 6+ to run elm-live natively.

Sorry for the surprise – and let me know if it helped 🙂

**Comment by clarkware** [comment]
Thanks for the quick response, @tomekwi. I was getting the error using Node v5.4.0. Upgrading to Node 6 indeed fixed it. Thanks!

**Comment by tomekwi** [comment]
To prevent such things in the future, I just released a patch fixing the `engines` field in the `package.json`. Unfortunately, npm has some [convoluted logic](http://www.marcusoft.net/2015/03/packagejson-and-engines-and-enginestrict.html#using-the-engines-configurations) which causes that setting to be ignored, but yarn does the job perfectly:

```sh
$ yarn global add elm-live
yarn global v0.16.1
warning No license field
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error elm-live@2.6.2: The engine "node" is incompatible with this module. Expected version ">= 6.0.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/global for documentation about this command.
```

**Comment by tomekwi** [comment]
Fixed in 2.6.2 🙂
